openpype/pipeline/anatomy.py

```diff
@@ -908,7 +928,17 @@ def _keys_to_dicts(self, key_list, value):
             return {key_list[0]: value}
         return {key_list[0]: self._keys_to_dicts(key_list[1:], value)}
 
-    def _rootless_path(self, result, final_data):
+    @classmethod
+    def rootless_path_from_result(cls, result):
+        """Calculate rootless path from formatting result.
+
+        Args:
+            result (StringTemplate): Result of StringTemplate formatting.
```

Comment:
Shouldn't the type here be `TemplateResult`?

Reply:
```suggestion
            result (TemplateResult): Result of StringTemplate formatting.
```